src/providers/LIFIProvider.tsx

```diff
@@ -0,0 +1,26 @@
+import LIFI from "@lifi/sdk";
+import React, { createContext, useContext, useMemo } from "react";
+
+let lifi: LIFI;
+
+const LIFIContext = createContext<LIFI>(null!);
+
+export const useLiFi = (): LIFI => useContext(LIFIContext);
+
+export const LIFIProvider: React.FC<React.PropsWithChildren> = ({
+  children,
+}) => {
+  const value = useMemo(() => {
+    const config = {};
+    if (!lifi) {
+      lifi = new LIFI({
+        disableVersionCheck: true,
```

Comment:
When enabled, does this property make you register the SDK/API version?

Reply:
I'm not entirely sure on this will have to check with LIFI, the config parameter is completely optional I just left it here in case we wanted to edit this part and customise it later.